scipy/sparse/linalg/_isolve/_gcrotmk.py

```diff
@@ -242,6 +242,11 @@ def gcrotmk(A, b, x0=None, *, tol=1e-5, maxiter=1000, M=None, callback=None,
         smallest singular values using the scheme discussed in [1,2].
         See [2]_ for detailed comparison.
         Default: 'oldest'
+    tol : float, optional, deprecated
+
+        .. deprecated:: 1.12.0
+           `gcrotmk` keyword argument `tol` is deprecated in favor of `rtol`
+           and will be removed in SciPy 1.14.
```

Comment:
The methods which already have the deprecation say "1.14.0" instead of "1.14", in case you want to change those  for consistency.